drake/bindings/matlab/+autodiffutils/newAutoDiff.m

```diff
@@ -0,0 +1,18 @@
+function obj = newAutoDiff(value, derivatives)
```

Comment:
it's actually "new" because it invokes a constructor, but I see your point. I'll rename it to something clearer.